src/components/taskDetails/Details.tsx

```diff
@@ -31,6 +33,13 @@ const Details: FC<TaskDetailsProps> = ({ detailType, value }) => {
                     <>{isGitHubLink ? 'N/A' : value ?? 'N/A'}</>
                 )}
             </span>
+            <span>
+                {detailType === 'Ends On' && url && (
```

Comment:
- can we not just rely on `detailsType`? also can we take a flag to show/hide this icon like it was before?

Reply:
this icon will be shown only in `ends on` row.